reference/test/matrix/batch_csr_kernels.cpp

```diff
@@ -201,6 +201,27 @@ TYPED_TEST(Csr, CanTwoSidedScaleWithDifferentValues)
 }
 
 
+TYPED_TEST(Csr, CanAddScaledIdentity)
```

Comment:
If you feel like it, you could add a test for a matrix that doesn't have a diagonal (same for ell)